### ISSUE TITLE
TS lint stopped working.

### ISSUE THREAD

**Issue by IlyaKozik** [Microsoft/vscode-tslint#263]
I have the following configuration:
```
{
    "extends": [
        "tslint:recommended"
    ],
    "rulesDirectory": [
        "node_modules/codelyzer"
    ],
    "rules": {
        "indent": [
            true,
            "spaces",
            4
        ],
        // Custom
        "trailing-comma": [
            false,
            {
                "multiline": "always",
                "singleline": "never"
            }
        ],
        "interface-name": [
            false,
            "always-prefix"
        ],
        // General
        "no-console": [
            true,
            "time",
            "timeEnd",
            "trace"
        ],
        "no-string-literal": false,
        "object-literal-sort-keys": false,
        "ordered-imports": [
            true,
            {
                "import-sources-order": "lowercase-last",
                "named-imports-order": "lowercase-first"
            }
        ],
        "quotemark": [
            true,
            "single",
            "avoid-escape"
        ],
        "variable-name": [
            true,
            "allow-leading-underscore",
            "allow-pascal-case",
            "ban-keywords",
            "check-format"
        ],
        // Angular codelyzer rules
        // Angular functionality
        "banana-in-box": true,
        "invoke-injectable": true,
        "no-access-missing-member": true,
        "pipe-impure": true,
        "templates-no-negated-async": true,
        "templates-use-public": true,
        "template-to-ng-template": true,
        // Angular maintainability
        "no-attribute-parameter-decorator": true,
        "no-forward-ref": false,
        "no-input-rename": true,
        "no-output-rename": true,
        "use-life-cycle-interface": true,
        "use-pipe-decorator": true,
        "use-pipe-transform-interface": true,
        "use-view-encapsulation": true,
        // Angular style
        "angular-whitespace": [
            true,
            "check-interpolation"
        ],
        "component-class-suffix": [
            true,
            "Component"
        ],
        "component-selector": [
            true,
            "element",
            "idm",
            "kebab-case"
        ],
        "directive-class-suffix": [
            true,
            "Directive"
        ],
        "directive-selector": [
            true,
            "attribute",
            "idm",
            "camelCase"
        ],
        "import-destructuring-spacing": true,
        "pipe-naming": [
            true,
            "camelCase",
            "idm"
        ],
        "use-host-property-decorator": true,
        "use-input-property-decorator": true,
        "use-output-property-decorator": true
    }
}
```

**Comment by egamma** [comment]
@IlyaKozik  can you please follow the trouble shooting guide and add a trace as described here:
https://github.com/Microsoft/vscode-tslint/tree/master/tslint#faq to the issue.

**Comment by zubivan** [comment]
@IlyaKozik what was the issue for you?
I also experienced failures with the latest update, what helped was removing `tslint.rulesDirectory` from vscode settings and introducing it in tslint.config.

**Comment by egamma** [comment]
@IlyaKozik do you define tslint.nodePath with a relative path in your settings? Relative paths in the settings for the rulesDirectory and the nodePath were broken, but this has been fixed in vscode-tslint 1.0.8. Can you please try whether you can reproduce the problem with this version.

**Comment by IlyaKozik** [comment]
@egamma yes, it was the issues with relative paths. And, yes, it has been fixed in vscode-tslint 1.0.8. Thanks.
